CHANGELOG.md

```diff
@@ -1,3 +1,86 @@
+* Add `"mobile: activeAppInfo"` feature [#1025](https://github.com/appium/appium-xcuitest-driver/pull/1025)
+* Settings
+  * `mjpegScalingFactor` [#983](https://github.com/appium/appium-xcuitest-driver/pull/983)
+* New capabilities
+  * `includeSafariInWebviews` [#1060](https://github.com/appium/appium-xcuitest-driver/pull/1060)
+  * `safariGlobalPreferences` [#1057](https://github.com/appium/appium-xcuitest-driver/pull/1057)
+* Remove some third-party dependencies
+  * `idevicedate` [#1042](https://github.com/appium/appium-xcuitest-driver/pull/1042)
+  * `iproxy` [#996](https://github.com/appium/appium-xcuitest-driver/pull/996)
+  * `idevicesyslog` [#1000](https://github.com/appium/appium-xcuitest-driver/pull/1000)
+  * `idevicelocation` [#1006](https://github.com/appium/appium-xcuitest-driver/pull/1006)
+  * `ifuse` [#1019](https://github.com/appium/appium-xcuitest-driver/pull/1019)
+  * `ios-webkit-debug-proxy` [#1023](https://github.com/appium/appium-xcuitest-driver/pull/1023)
+* Get only user apps in file-movement for performance aspect [#1014](https://github.com/appium/appium-xcuitest-driver/pull/1014)
+* Switch `reduceMotion` via settings API instead of update simulator pref [#1065](https://github.com/appium/appium-xcuitest-driver/pull/1065)
+* Add the necessary primitives to be able to automate split-screen apps [#209](https://github.com/appium/webdriveragent/pull/209)
+* Add a new endpoint for element rotation [#213](https://github.com/appium/webdriveragent/pull/213)
+* Add `processArguments` and other useful information in `"mobile: deviceInfo"` [#194](https://github.com/appium/webdriveragent/pull/194) [#210](https://github.com/appium/webdriveragent/pull/210)
+* Consider active application based on the current screen element [#214](https://github.com/appium/webdriveragent/pull/214)
+* Make the coordinates of the screen point configurable [#215](https://github.com/appium/webdriveragent/pull/215)
+* Allow setting compression quality and scaling factor for `mjpeg-stream` [#196](https://github.com/appium/WebDriverAgent/pull/196)
+* Fix `GET` `/timeouts` [#1067](https://github.com/appium/appium-xcuitest-driver/pull/1067)
+* Switch `async execute` to not need CORS (works for real and simulated devices) [#1063](https://github.com/appium/appium-xcuitest-driver/pull/1063)
+* Fix uninstalling of `WDA` with '.xctrunner' suffix for real device [#1052](https://github.com/appium/appium-xcuitest-driver/pull/1052)
+* Fix caching issues [#1053](https://github.com/appium/appium-xcuitest-driver/pull/1053)
+* Fix `mjpegstream` logic for returning screenshots [#1039](https://github.com/appium/appium-xcuitest-driver/pull/1039)
+* Fix Node bug when ending session [#1040](https://github.com/appium/appium-xcuitest-driver/pull/1040)
+* Fix launching `WDA` multiple times [#999](https://github.com/appium/appium-xcuitest-driver/pull/999)
+* Fix "get active element" [#1011](https://github.com/appium/appium-xcuitest-driver/pull/1011)
```

Comment:
the fix about getting the list of active contexts

Reply:
What do you mean? It has nothing to do with contexts.

Reply:
oh, I checked the wrong link. all good